symfit/core/models.py

```diff
@@ -1100,7 +1103,7 @@ def eval_components(self, *args, **kwargs):
             initial_dependent,
             t_bigger,
             args=tuple(
-                bound_arguments.arguments[param.name] for param in self.params),
+                bound_arguments.arguments[param.name] for param in self.model_params),
```

Comment:
why was this switch needed? I'm guessing because the parameter corresponding to the initial value should not be provided here? I feel like this needs either a comment or a better variable name.

Reply:
It was needed because `a0` should not be passed to the python version of the component, to be integrated by scipy. Instead, `a0` should be passed to scipy in `initial_dependent`.
I'll add a comment